Storage/SQL/BrowserDB.swift

```diff
@@ -77,7 +77,7 @@ open class BrowserDB {
     // appear in a query string.
     open static let MaxVariableNumber = 999
 
-    public init(filename: String, secretKey: String? = nil, files: FileAccessor) {
+    public init(filename: String, secretKey: String? = nil, files: FileAccessor, leaveClosed: Bool = false) {
```

Comment:
extremely minor nit: related to my other comment below, maybe a more accurate name for this would be `skipSchemaUpdate`?